www/cgi-bin/NETweb.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import os, sys
+import os.path as path
+
+# The environment variable NETLIBDIR can be used to override the automatic
+# path setup.
+NETLIBDIR = os.environ.get("NETLIBDIR")
+if NETLIBDIR == None or NETLIBDIR.strip() == "":
+    NETLIBDIR = path.realpath(path.join(path.dirname(__file__), '..', '..'))
+sys.path.append(NETLIBDIR)
```

Comment:
This path setup defaults to assuming `NETlib.py` will live two parent directories up, which is appropriate for the git directory layout. Currently this could be overridden by setting the environment variable `NETLIBDIR` with a different value, but it may be better to use a configuration file on disk instead.